src/check/canonicalize.zig

```diff
@@ -141,8 +198,9 @@ pub fn canonicalize_file(
 
     // canonicalize_header_packages();
 
-    // Track the start of scratch defs
+    // Track the start of scratch defs and statements
     const scratch_defs_start = self.can_ir.store.scratchDefTop();
+    const scratch_statements_start = self.can_ir.store.scratch_statements.top();
```

Comment:
I wasn't sure about this, but currently we're only printing out the "defs" in a module, and I needed the statements to see type declarations.